docs/go.mod

```diff
@@ -1,9 +1,3 @@
 module github.com/distribution/distribution/docs
 
-go 1.21.1
-
-require (
-	github.com/google/docsy v0.7.1 // indirect
-	github.com/imfing/hextra v0.5.0 // indirect
-	github.com/thegeeklab/hugo-geekdoc v0.41.2 // indirect
-)
+go 1.20
```

Comment:
We could even remove the version, but looks like go mod tidy wants to add "something" back. Technically "any" version would work, so we could set it to something really low.